docs/models/job.md

```diff
@@ -0,0 +1,35 @@
+# OneBlink .Net SDK | Job Model
+
+
+## `Job()`
+
+### Constructor
+
+| Parameter           | Required | Type                          | Description                               | Default Value |
+| ------------------- | -------- | ----------------------------- | ----------------------------------------- | ------------- |
+| `username`          | No       | `string`                      | Username associated with the job.         | null          |
+| `formId`            | No       | `int`                         | formId associated with the job.           | null          |
+| `details`           | No       | [`JobDetail`](<#JobDetail()>) | Instance of the job's details.            | null          |
+| `externalId`        | No       | `string`                      | external Id associated with the job.      | null          |
+| `prefillFormDataId` | No       | `string`                      | prefillFormDataId associated with the job | null          |
+
+### Example
+
+```c#
+OneBlink.SDK.Model.Job jobToCreate = new OneBlink.SDK.Model.Job(
+  username: "developer@oneblink.io",
+  formId: 4,
+  details: jobToCreateDetails,
+  externalId: "MyExternalId",
+  prefillFormDataId: "MyPrefillFormDataId"
```

Comment:
This should be in the extra properties, we dont want our customers setting this value.

Reply:
fixed.